cranelift-native/Cargo.toml

```diff
@@ -14,7 +14,7 @@ cranelift-codegen = { path = "../cranelift-codegen", version = "0.45.0", default
 target-lexicon = "0.8.1"
 
 [target.'cfg(any(target_arch = "x86", target_arch = "x86_64"))'.dependencies]
-raw-cpuid = "6.0.0"
+raw-cpuid = "7.0"
```

Comment:
This is the only non semver compatible version bump.

Reply:
The tests are passed. Need it downcast to 6.0 or 6.0.0 ?